src/poetry/installation/chef.py

```diff
@@ -42,10 +46,6 @@ def is_wheel(self, archive: Path) -> bool:
         return archive.suffix == ".whl"
 
     def get_cached_archive_for_link(self, link: Link) -> Link:
-        # If the archive is already a wheel, there is no need to cache it.
-        if link.is_wheel:
-            return link
```

Comment:
Also whl was downloaded every poetry install, but at the same time saved to the cache folders.
So the cache exists but was useless.
So I removed this lines.